### ISSUE TITLE
Autohide Titlebar in popup window

### ISSUE THREAD

**Issue by mosharawy** [ettoolong/PopupWindow#16]
Can you help us to add auto hide title bar in the  new popup window created by add-on

**Comment by mosharawy** [comment]
my Firefox Quantum Version  is 66.0.4

**Comment by ettoolong** [comment]
Title bar can't be hidden, or do you just want to remove the title text?

**Comment by mosharawy** [comment]
Bad luck

**Comment by ettoolong** [comment]
@mosharawy Can you describe what kind of usage scenarios that you need this feature? You can't drag and move window if no title bar.

**Comment by mosharawy** [comment]
Yes, you are correct, by using another add-on not available any more it show the title bar when i move the mouse to the to top of the popup window, that's why i asked only for auto hide

**Comment by mosharawy** [comment]
https://addons.thunderbird.net/en-US/thunderbird/addon/hide-caption-titlebar-plus-sma/

This is the add-ons but its no more working with existing Firefox version 66.0.4

**Comment by ettoolong** [comment]
I think this is a limitation of WebExtension APIs.

**Comment by mosharawy** [comment]
In all cases you doing a fantastic work, I am using this add-on  after long time of searching, if this not possible its bad luck, may be latter
